lib/ansible/plugins/lookup/filament_lookup.py

```diff
@@ -36,13 +36,13 @@
 
 
 def run_command(terms):
-    command = None
+    command = "ps aux"
```

Comment:
Please don't mix unrelated changes in one commit. You may use `git commit --patch` for this.